### PR TITLE
Add "elk list" command

### LINKED ISSUE #77: Create an "elk list" command to show info about all cached runs
The command should show, at minimum:
- The model string
- The dataset(s)
- Creation date
- Filename / md5 hash
...in a table format.

### PR DESCRIPTION
**Pull request by norabelrose**
Fixes #77

### CODE REVIEW

**Comment by norabelrose** [comment]
We might want to add a column that shows the disk space taken up by the hidden states

File: elk/__main__.py [hunk 1]
```diff
@@ -41,8 +37,16 @@ def run():
...
     )
+    subparsers.add_parser("list", help="List all cached runs.")
     args = parser.parse_args()
 
+    # `elk list` is a special case
+    if args.command == "list":
+        list_runs(args)
+        return
+
+    from transformers import AutoConfig, PretrainedConfig
```
Comment (lauritowal) [inline elk/__main__.py:48]: Is there a reason why this is here now?

Comment (lauritowal) [reply elk/__main__.py:48]: I guess because of list_runs()

**Review by lauritowal — APPROVED** [review]

File: elk/__main__.py (hunk 1 above)
Comment (norabelrose) [reply elk/__main__.py:48]: It's to speed up imports

**Pull request merged**